### PR TITLE
Refactored dynamic_voting_window()

### PR DESCRIPTION
**Pull request by hongaar**
[Model on Google Drive](https://docs.google.com/spreadsheets/d/1qNMzaWe_cSjncqgm3HKy5UIb8LdhNgUm916OXZqO_4s/edit?usp=sharing)

Not perfect, but at least we use `log()` now ;)

### CODE REVIEW

File: github_api/misc.py [hunk 1]
```diff
@@ -1,3 +1,4 @@
+from math import log
```
Comment (mdcfe) [inline github_api/misc.py:2]: There needs to be another line here for Travis not to fail.

**Comment by phil-r** [comment]
why everyone loves `log` so much? [Sigmoid functions](https://en.wikipedia.org/wiki/Sigmoid_function) are way cooler imho 🤓

**Comment by hongaar** [comment]
@phil-r looks better suited for the job indeed. gotta run now but feel free to edit my fork.

**Comment by phil-r** [comment]
@hongaar we can modify later, your solution is also cool ;)

**Comment by mark-i-m** [comment]
I like logs. Where else would I look to find out what caused my crash? :stuck_out_tongue:

**Pull request merged**